### PR TITLE
Move some namespaces on model

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets |
| License         | MIT

### CODE REVIEW

File: src/Bundle/EventListener/ODMTranslatableListener.php [hunk 1]
```diff
@@ -68,11 +70,11 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
             return;
         }
 
-        if ($reflection->implementsInterface(TranslatableInterface::class)) {
+        if ($reflection->implementsInterface(TranslatableInterface::class) || $reflection->implementsInterface(LegacyTranslatableInterface::class)) {
```
Comment (loic425) [inline src/Bundle/EventListener/ODMTranslatableListener.php]: Hum, I'm not sure we need that.

File: tests/Application/src/Entity/LegacyBook.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    use TranslatableTrait {
+        __construct as private initializeTranslationsCollection;
+    }
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private int $id;
+
+    public function getId(): int
```
Comment (diimpp) [inline tests/Application/src/Entity/LegacyBook.php]:
```suggestion
    private ?int $id = null;

    public function getId(): ?int
```

Current typehint will throw exception on call on new entity.

Comment (diimpp) [reply tests/Application/src/Entity/LegacyBook.php]: Would be cool to sync code with regular Book/BookTranslation

> tests/Application/src/Entity/Book.php

**Review by diimpp — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @loic425!

**Pull request merged**